Android App/app/fabric.properties

```diff
@@ -1,3 +0,0 @@
-#Contains API Secret used to validate your application. Commit to internal source control; avoid making secret public.
-#Wed Jan 10 13:50:07 PKT 2018
-apiSecret=1c0e86b595e8236899f8105cae1bd151d6fe29e19f42a19bca80b7fe0d807582
```

Comment:
This is exactly one of those things which should never be pushed to remote which is public :)

Reply:
agreed and we discussed with Shazaib this too a couple of days before thanks for doing that.